app-information.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Web App Manifest - App Information
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class='remove'>
+    var respecConfig = {
+      mdn: true,
+      previousPublishDate: "2013-12-17",
+      specStatus: "NOTE",
```

Comment:
We will switch to a note when we publish. 

```suggestion
      specStatus: "ED", 
```